src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
Aren't you supposed to be the law expert here? 😆

The rule I heard was having both - Mono & .NET Foundation at the top of the file. Again, we may need to check. `HttpListener` code came from Mono, so you might want to check there.